app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -137,6 +137,10 @@ class SettingsFragment : PreferenceFragmentCompat(), AccountObserver {
         val appName = getString(R.string.app_name)
         aboutPreference?.title = getString(R.string.preferences_about, appName)
 
+        findPreference<Preference>(getPreferenceKey(R.string.pref_key_add_private_browsing_shortcut))?.apply {
+            isVisible = !PrivateShortcutCreateManager.doesPrivateBrowsingPinnedShortcutExist(requireContext())
```

Comment:
nit: Preference already has a context getter
```suggestion
            isVisible = !PrivateShortcutCreateManager.doesPrivateBrowsingPinnedShortcutExist(context)
```